docs/configuration/config-modification.md

```diff
@@ -28,61 +45,61 @@ interface ConfigInterface {
   This field contains configuration responsible for rendering specific parts of the AsyncApi component.
   All fields are set to `true` by default.
 
-- **disableDefaultTheme?: boolean**
+- **expand?: Partial<ExpandConfig>**
 
-  This field informs whether the forwarded [`ThemeInterface`](../../library/src/theme/theme.ts#L5) should be concatenated with the [default theme](../../library/src/theme/default.ts) or if the AsyncApi component should consider only the forwarded one.
-  This field is set to `false` by default.
-
-  > **NOTE:** When you set this flag to `true`, you must provide definitions of all styles.
+  This field contains configuration responsible for initial expanding specific parts of the AsyncApi component.
+  `root` means root component for specific parts of the AsyncApi component. `Elements` means elements inside `root` component.
+  By default `expand.channels.root` is set to `true`.
 
 - **showErrors?: boolean**
 
   This field turns on or off the option displaying validation or parsing errors that show at the top of the component.
-  This field is set to `false` by default.
+  This field is set to `true` by default.
+
+- **parserOptions?: ParserOptions**
+
+  This field contains configuration for [`asyncapi-parser`](https://github.com/asyncapi/parser). See available options [here](https://github.com/asyncapi/parser-js/blob/master/API.md#parser).
+  This field is set to `null` by default.
 
 ## Examples
 
-See exemplary component configuration in JavaScript and TypeScript.
+See exemplary component configuration in TypeScript and JavaScript.
 
-### JavaScript
+### TypeScript
 
-```jsx
+```tsx
 import * as React from "react";
 import { render } from "react-dom";
-import AsyncApiComponent from "asyncapi-react";
+import AsyncApiComponent, { ConfigInterface } from "asyncapi-react";
 
 import { schema } from "./mock";
 
-const config = {
+const config: Partial<ConfigInterface> = {
   show: {
-    security: false,
     schemas: false
   },
-  disableDefaultTheme: true,
-  showErrors: true
+  showErrors: false
 };
 
 const App = () => <AsyncApiComponent schema={schema} config={config} />;
 
 render(<App />, document.getElementById("root"));
 ```
 
-### TypeScript
+### JavaScript
```

Comment:
I know we love TS, but pure JS example should be first, as it was before 😢

Reply:
Nope :P